container_workflow_tool/main.py

```diff
@@ -426,12 +433,10 @@ def list_images(self):
 
     def print_upstream(self):
         """Prints the upstream name and url for images used in config"""
-        template = "{component} {img_name} {ups_name} {url}"
         for i in self._get_images():
-            ups_name = re.search(".*\/([a-zA-Z0-9-]+).git",
+            ups_name = re.search(".*\\/([a-zA-Z0-9-]+).git",
```

Comment:
Not sure I about the change here. I think the `\` was used as an escape for `/`. Though now I am not sure if escaping `/` is even necessary. If not I would just remove the `\` entirely.